app/Main.hs

```diff
@@ -424,38 +423,36 @@ checkForUpdates applyMinorUpdates applyMajorUpdates = do
     isMinorReleaseFrom (x : xs) (y : ys) = y == x && ys > xs
     isMinorReleaseFrom _        _        = False
 
-verify :: String -> IO ()
-verify inputName = case mkPackageName (pack inputName) of
-  Left pnError -> echoT . pack $ "Error while parsing input package name: " <> show pnError
-  Right name -> do
+verify :: Maybe Text -> Maybe Text -> IO ()
```

Comment:
I think this should be `Either Text (Maybe Text) -> IO ()` (possibly using an actual ADT instead) since `--after` is not valid when a package is specified. It should be possible to have `optparse-applicative` recognize only the valid combinations.